Cloud Application Manager/Automating Deployments/google-deployment-manager.md

```diff
@@ -86,10 +87,10 @@ while a different deployment still uses the very same resource. Other times the
 might get blocked, because it can not clean up a resource it has deployed,
 if a different instance has already deleted it.
 
-A good practice to avoid these scenarios is to prefix the name of each resource with the name of the deployment,
+A good practice to avoid these scenarios is to prefix the name of each resource with the **name of the deployment**,
 which happens to be the *service-id* in Cloud Application Manager.
 
-```
+```Jinja
```

Comment:
Would it be `yaml` best?

Reply:
Depends on the rendered style, but really nothing spectacular to see in Yaml, but with Jinja it highlights the variable substitutions, which is informative.